xml/obs_best_practice_kiwi_editor.xml

```diff
@@ -9,45 +9,165 @@
     </imageobject>
    </mediaobject>
   </figure>
-  <para> Modify the &kiwi; configuration as you want, and click the "Save" button
-   when you finish. </para>
- </sect2>
+  <itemizedlist>
+    <listitem>
+      <para>
+        <emphasis>Expert Mode:</emphasis> This mode provides you with a set of customizable parameters for creating a repository.
+      </para>
+      <itemizedlist>
+        <listitem>
+         <para>
+           <emphasis>Type:</emphasis> We is only allowed <emphasis>rpm-md</emphasis> and <emphasis>apt-deb</emphasis>.
+         </para>
+        </listitem>
+        <listitem>
+         <para>
+            <emphasis>Priority:</emphasis> Repository priority for this given repository.
+         </para>
+        </listitem>
+        <listitem>
+         <para>
+            <emphasis>Alias:</emphasis> Alternative name for the configured repository.
+         </para>
+        </listitem>
+        <listitem>
+         <para>
+            <emphasis>Source Path:</emphasis> Define the repository path
+         </para>
+        </listitem>
+        <listitem>
+         <para>
+            <emphasis>User:</emphasis> Specifies a user name for the given repository.
+         </para>
+        </listitem>
+        <listitem>
+         <para>
+            <emphasis>Password:</emphasis> Specifies a password for the given repository.
+         </para>
+        </listitem>
+        <listitem>
+         <para>
+            <emphasis>Prefer License:</emphasis> The repository providing this attribute will be used primarily to install
+            the license tarball if found on that repository.
+         </para>
+        </listitem>
+        <listitem>
+         <para>
+            <emphasis>Image Include:</emphasis> Specifies whether the given repository should be configured as a repository in the image
+         </para>
+        </listitem>
+        <listitem>
+         <para>
+            <emphasis>Replaceable:</emphasis> This attribute should only be applied in the context of a boot image description.
+            The specified repository may be replaced by the repositories specified in the image description.
+         </para>
+        </listitem>
+      </itemizedlist>
+    </listitem>
+  </itemizedlist>
+  <figure>
+   <title>&kiwi; Adding a new repository - Expert Mode</title>
+   <mediaobject>
+    <imageobject>
+     <imagedata fileref="kiwi_editor_repo_expert.png"/>
+    </imageobject>
+   </mediaobject>
+  </figure>
+
+  <para> If you want to use the configuration of the current project you only need to
+    check <emphasis>Use project repositories</emphasis> checkbox.</para>
+  <figure>
+   <title>&kiwi; Use project configuration</title>
+   <mediaobject>
+    <imageobject>
+     <imagedata fileref="kiwi_editor_use_project_repositories.png"/>
+    </imageobject>
+   </mediaobject>
+  </figure>
+  <note> <para> This option will remove from you kiwi file the other repositories that you have set previously. </para></note>
```

Comment:
Typo: from your

And I think we should slightly rephrase the sentence: `will remove the other repositories from your kiwi file `. The other versions a bit strange (too formal?) to me.